plugins/org.wso2.developerstudio.eclipse.esb.docker/src/org/wso2/developerstudio/eclipse/esb/docker/wizard/ExportAndGenerateDockerImageWizard.java

```diff
@@ -0,0 +1,320 @@
+
+            if (mainPage.getServerRoleList().containsKey(artifactInfo)) {
+                properties.put(artifactInfo, serverRoleList.get(artifactInfo));
+            } else {
+                properties.put(artifactInfo, "capp/ApplicationServer");
+            }
+        }
+
+        properties.put("artifact.types", artifactTypeMapping.getArtifactTypes());
+        parentPrj.getModel().setProperties(properties);
+    }
+
+    private Properties identifyNonProjectProperties(Properties properties) {
+        Map<String, DependencyData> dependencies = projectList;
+
+        for (Iterator<DependencyData> iterator = dependencies.values().iterator(); iterator.hasNext();) {
+            DependencyData dependency = (DependencyData) iterator.next();
+            String artifactInfoAsString = DistProjectUtils.getArtifactInfoAsString(dependency.getDependency());
+
+            if (properties.containsKey(artifactInfoAsString)) {
+                properties.remove(artifactInfoAsString);
+            }
+        }
+
+        // Removing the artifact.type
+        properties.remove("artifact.types");
+
+        return properties;
+    }
+
+    public void addPages() {
+        if (!initError) {
+            addPage(detailsPage);
+            addPage(mainPage);
+            super.addPages();
+        }
+    }
+
+    private void setSessionProperty() {
+        try {
+            detailsPage.getSelectedProject()
+                    .setSessionProperty(
+                            new QualifiedName(ExportImageWizardConstants.EMPTY_STRING,
+                                    detailsPage.getSelectedProject().getName()),
+                            detailsPage.getTxtExportPathText().getText());
+        } catch (CoreException e) {
+            log.error("Error geting session properties", e);
+        }
+    }
+
+    private int openMessageBox(Shell shell, String title, String message, int style) {
+        MessageBox exportMsg = new MessageBox(shell, style);
+        exportMsg.setText(title);
+        exportMsg.setMessage(message);
+
+        return exportMsg.open();
+    }
+
+    private String getWorkingDirectory() {
+        String workingDirectory = null;
+        String OS = System.getProperty(ExportImageWizardConstants.OS_NAME,
+                ExportImageWizardConstants.SYSTEM_PROPERTY_TYPE_GENERIC).toLowerCase(Locale.ENGLISH);
+
+        if ((OS.indexOf(ExportImageWizardConstants.OS_TYPE_MAC) >= 0)
+                || (OS.indexOf(ExportImageWizardConstants.OS_TYPE_DARWIN) >= 0)) {
+            String eiToolingHomeForMac = ExportImageWizardConstants.EI_TOOLING_HOME_MACOS;
+            File macOSEIToolingAppFile = new File(eiToolingHomeForMac);
+
+            if (macOSEIToolingAppFile.exists()) {
+                workingDirectory = eiToolingHomeForMac;
+            } else {
+                java.nio.file.Path path = Paths.get(ExportImageWizardConstants.EMPTY_STRING);
```

Comment:
Import the package.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/800